### PR TITLE
Added: Attendance Tracker Next.js

### PR DESCRIPTION
**Pull request by anisharma07**
Closes:  #317

- **Title** :  Attendance Tracker
- **Name:** :  Anirudh Sharma
- **Idenitfy yourself:** : GSSOC'24 Contributor

### Describe the add-ons or changes you've made 📃
Added the Project folder to the Nextjs/advanced folder
updated nextjs/ README.md

### How Has This Been Tested? ⚙️
-> The app has been tested both on mobile and computer under different situations and taking different values
-> Detailed working video of web app is attached.

### Screenshots 📷

image

image

https://github.com/user-attachments/assets/c15f3199-d656-4c38-b64d-0251a2ddd400

### Type of change ☑️

- [x] My code follows the code style of this project.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

### Checklist: ☑️
- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄
Please assign under Gssoc'24 & mark appropriate labels ....... a level 3 label would be helpful (created whole app from scratch from UI designing to building backend)

### CODE REVIEW

File: Next-JS-Projects/Advanced/Attendance-Tracking-App/README.md
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/Advanced/Attendance-Tracking-App/README.md:1]: Follow the readme template and attach 1 file of screenshot in `webp` format and named it to `screenshot` i.e. `screenshot.webp`

File: Next-JS-Projects/README.md
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/README.md:1]: Don't update any other thing else enlisting your project here.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Next-JS-Projects/Advanced/Attendance-Tracking-App/README.md
Comment (anisharma07) [reply Next-JS-Projects/Advanced/Attendance-Tracking-App/README.md:1]: done updating the readme file & adding screenshot.webp

File: Next-JS-Projects/README.md
Comment (anisharma07) [reply Next-JS-Projects/README.md:1]: project is updated in the list

**Comment by anisharma07** [comment]
@Avdhesh-Varshney 
Done updating readme & adding screenshot & updated the projects list

**Comment by anisharma07** [comment]
@Avdhesh-Varshney 
Please review the Changes ... today is last date for Gssoc'24

File: Next-JS-Projects/README.md
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/README.md:1]: Remoke your un-necessary changes.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by anisharma07** [comment]
@Avdhesh-Varshney 
Please review the PR

**Review by Avdhesh-Varshney — APPROVED** [review]

**Pull request merged**